### PR TITLE
Remove CentOS 6, CentOS 8 and Debian 9 unsupported clients

### PR DESCRIPTION
**Pull request by Bischoff**
Remove CentOS 6 and Debian 9 from main.tf's (the first one has no impact on this specific PR).

Also adapt the tests on PRs so they use `EL_8` instead of `CentOS_8`. Taking the occasion to rename the `CENTOS_CLIENT` variables to something not CentOS-related, and same for Ubuntu.

Issue: https://github.com/SUSE/spacewalk/issues/20607

### CODE REVIEW

**Review by ktsamis — APPROVED** [review]

File: jenkins_pipelines/environments/manager-4.2-qe-build-validation [hunk 1]
```diff
@@ -35,7 +35,7 @@ node('sumaform-cucumber-provo') {
...
...        text(name: 'custom_repositories', defaultValue: '{"server":{"server_stack":"","salt15sp2_base":"","salt15sp2_python2_module":"","salt15sp2_server_apps_module":"","extra_sle_that_affects_suma_srv":""},"proxy":{"salt":"","tools":"","salt15sp2":"","extra_sle_that_affects_suma_proxy":""},"sle12sp4_minion":{"salt":"","tools":""},"sle12sp4_client":{"tools":""},"sle12sp5_minion":{"salt":"","tools":""},"sle12sp5_client":{"tools":""},"sle15_minion":{"salt":"","tools":""},"sle15_client":{"tools":""},"sle15sp1_minion":{"salt":"","tools":"","python":"","server_apps":""},"sle15sp1_client":{"tools":""},"sle15sp2_minion":{"salt":"","tools":"","python":"","server_apps":""},"sle15sp2_client":{"tools":""},"sle15sp3_minion":{"salt":"","tools":"","python":"","server_apps":""},"sle15sp3_client":{"tools":""},"debian10_minion":{"salt":"","tools":""},"ubuntu1804_minion":{"salt":"","tools":""},"ubuntu2004_minion":{"salt":"","tools":""},"ceos7_minion":{"salt":"","tools":""},"ceos7_client":{"tools":""},"rocky8_minion":{"salt":"","tools":""},"rocky8_client":{"tools":""},"opensuse154arm_minion":{"salt":"","tools":""}}', description: 'Salt & Client Tools MU Repositories for each client, in json format')
```
Comment (srbarrios) [inline jenkins_pipelines/environments/manager-4.2-qe-build-validation:38]: So, CentOS 7 still supported? (I'm not sure)

File: jenkins_pipelines/environments/common/pipeline-pull-request.groovy [hunk 2]
```diff
@@ -258,7 +258,7 @@ def run(params) {
...
+                        sh "set +x; source /home/jenkins/.credentials set -x; export TF_VAR_SLE_CLIENT_REPO=${SLE_CLIENT_REPO};export TF_VAR_RHLIKE_CLIENT_REPO=${REDHAT_CLIENT_REPO};export TF_VAR_DEBLIKE_CLIENT_REPO=${DEBLIKE_CLIENT_REPO};export TF_VAR_OPENSUSE_CLIENT_REPO=${OPENSUSE_CLIENT_REPO};export TF_VAR_PULL_REQUEST_REPO=${PULL_REQUEST_REPO}; export TF_VAR_MASTER_OTHER_REPO=${MASTER_OTHER_REPO};export TF_VAR_MASTER_SUMAFORM_TOOLS_REPO=${MASTER_SUMAFORM_TOOLS_REPO}; export TF_VAR_TEST_PACKAGES_REPO=${TEST_PACKAGES_REPO}; export TF_VAR_MASTER_REPO=${MASTER_REPO};export TF_VAR_UPDATE_REPO=${UPDATE_REPO};export TF_VAR_ADDITIONAL_REPO_URL=${ADDITIONAL_REPO_URL};export TF_VAR_CUCUMBER_GITREPO=${cucumber_gitrepo}; export TF_VAR_CUCUMBER_BRANCH=${cucumber_ref}; export TERRAFORM=${terraform_bin}; export TERRAFORM_PLUGINS=${terraform_bin_plugins}; ./terracumber-cli ${common_params} --logfile ${resultdirbuild}/sumaform.log ${env.TERRAFORM_INIT} --taint '.*(domain|main_disk).*' --runstep provision"
```
Comment (srbarrios) [inline jenkins_pipelines/environments/common/pipeline-pull-request.groovy:261]: Wasn't `RHLIKE_CLIENT_REPO` ? Here I see `REDHAT_CLIENT_REPO`

Comment (Bischoff) [reply jenkins_pipelines/environments/common/pipeline-pull-request.groovy:261]: OOPS !!!
thanks

File: jenkins_pipelines/environments/manager-4.2-qe-build-validation (hunk 1 above)
Comment (Bischoff) [reply jenkins_pipelines/environments/manager-4.2-qe-build-validation:38]: it is

Comment (nodeg) [reply jenkins_pipelines/environments/manager-4.2-qe-build-validation:38]: Yes. See https://www.suse.com/releasenotes/x86_64/SUSE-MANAGER/4.3/index.html#_unsupported_products

File: jenkins_pipelines/environments/common/pipeline-pull-request.groovy (hunk 2 above)
Comment (Bischoff) [reply jenkins_pipelines/environments/common/pipeline-pull-request.groovy]: Fixed. Good catch !

File: jenkins_pipelines/environments/manager-4.2-qe-build-validation (hunk 1 above)
Comment (Bischoff) [reply jenkins_pipelines/environments/manager-4.2-qe-build-validation:38]: And similar file for 4.2.

**Review by elariekerboull — APPROVED** [review]

**Pull request merged**